modules/apps/translation/translation-google-cloud-translator/src/main/java/com/liferay/translation/google/cloud/translator/internal/translator/GoogleCloudTranslator.java

```diff
@@ -92,7 +101,7 @@ public boolean isEnabled(long companyId) throws ConfigurationException {
 	public TranslatorPacket translate(TranslatorPacket translatorPacket)
 		throws PortalException {
 
-		if (!isEnabled()) {
+		if (!isEnabled(CompanyThreadLocal.getCompanyId())) {
```

Comment:
Add a groupId field to the translatorPacket, and use that here. Using threadlocals in this way is ok for deprecated methods and last resort hacks, but we should avoid it when possible.

Reply:
Sorry, I meant companyId, not groupId.